src/utils/wallet.ts

```diff
@@ -528,10 +535,14 @@ const wallet = {
       accessData.acctPathKey = encryptedAcctPathKey;
     }
 
-    if (authXpriv) {
-      // Auth path key will only be available if the provided key is a root key.
-      const encryptedAuthPathKey = encryptData(authXpriv.xprivkey, pin);
-      accessData.authKey = encryptedAuthPathKey;
+    if (authXpriv || derivedAuthKey) {
```

Comment:
Is it ok if we have both? Or should we protect in case the user passes the `authXpriv` as parameter with a root key?

Reply:
I don't think it should be an issue to have the auth key as a parameter and derived from the root key.
We should choose which one has priority, and I think having the argument as priority is the best choice.
What do you think?